src/aap_eda/settings/default.py

```diff
@@ -365,6 +366,7 @@ def _get_databases_settings() -> dict:
 REDIS_CLIENT_CACERT_PATH = settings.get("MQ_CLIENT_CACERT_PATH", None)
 REDIS_CLIENT_CERT_PATH = settings.get("MQ_CLIENT_CERT_PATH", None)
 REDIS_CLIENT_KEY_PATH = settings.get("MQ_CLIENT_KEY_PATH", None)
+REDIS_TLS = settings.get("MQ_TLS", False)
```

Comment:
Should default to None for backwards compatibility and respecting the provided, if any, value.

Reply:
I will comment in the thread below to keep the replies concise.